src/datastore/postgres-store.ts

```diff
@@ -72,6 +72,7 @@ export function getPgClientConfig(): ClientConfig {
     password: process.env['PG_PASSWORD'],
     host: process.env['PG_HOST'],
     port: parsePort(process.env['PG_PORT']),
+    ssl: process.env['PG_SSL'],
```

Comment:
I'm assuming the `pg` lib accepts a falsy value here, as in this is the equivalent of `ssl: process.env['PG_SSL'] ?? false`?

Reply:
Actually, I think this needs to have bit more logic to handle the string value `"false"` and/or `"0"` because technically those are truthy, non-empty strings.